test/scan/scanPerf.perfcompopts

```diff
@@ -0,0 +1 @@
+-smemFraction=4
```

Comment:
Nit, but I don't see any reason this needs to be a param. Could make it a config const and fold this into the perfexecopts.

Reply:
It was resulting in a divide by zero when it was not a param, though... why would that be?  Oh, I think it was before I guarded the computation by a conditional.  OK, I'm switching it back.